nemo/collections/nlp/modules/common/text_generation_utils.py

```diff
@@ -444,11 +546,19 @@ def switch(val1, val2, boolean):
 
 def forward_step(model, batch, tensor_shape):
 
+    # Should call MegatronGPTPPromptLearningModel's forward method
+    if hasattr(model, 'prompt_table'):
```

Comment:
change it to?
  ```python
  if isinstance(model, MegatronGPTPromptLearningModel):
  ```

Reply:
Oh, whoops. I thought I changed both of them.